### PR TITLE
closes #19 Multiple Yubikeys and mac dependencies

### LINKED ISSUE #19: Using two yubikeys not covered under guide
So, this might require more feedback, as am currently flummoxed.

If you want to use a Yubikey for Windows / MacOS / Linux login as part of a multi-factor authentication, then Yubico advises having a second Yubikey as backup.  This works fine, since I can use a Yubikey Nano for the Macbook Pro, and leave the desktop one plugged in.

However, if you try copying the GPG secret key and subkeys onto a second Yubikey, "keytocard" will not let you.  You'll get "secret key already stored on a card" and other similar warnings.  Is it not a good idea to use two Yubikeys?

### PR DESCRIPTION
**Pull request by EliasGabrielsson**
* Adding part descriping how to use multiple Yubikey with same keyset.
* Changed from `ykpers` to `ykman` as it is more feature complete.
* Added `pinentry-mac` which is preferable as it works with GUI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1201,6 +1202,27 @@ To require a touch for the signing and encrypting keys as well:
 
 The Yubikey will blink when it's waiting for the touch.
 
+## Using multiple Yubikeys for same GPG-keyset
+
+To use multiple Yubikeys (backup etc.) with same GPG-keyset repeat part [Transfer keys](#transfer-keys) for each additional key.
+When executing `gpg --card-status` the inserted Yubikeys serial number will be associated to the secret stub-keys. If another Yubikey with different serial number is used after that this error will be showned:
```
Comment (drduh) [inline README.md]: *sub*-keys

File: README.md [hunk 2]
```diff
@@ -1201,6 +1202,27 @@ To require a touch for the signing and encrypting keys as well:
...
+
+`Please insert the car with serialn number: [Assosiated serial number]`
```
Comment (drduh) [inline README.md]: Spelling.

File: README.md [hunk 3]
```diff
@@ -1201,6 +1202,27 @@ To require a touch for the signing and encrypting keys as well:
...
+
+The way around this is to remove the old assisiation by deleting the files holding it and run `gpg --card-status` again. To make this handy use the script below when connected a new Yubikey. The script need to be configured to match your key and having correct path to the `gpg` executeable. The key filename can be found by using `gpg --with-keygrip -K`, look for "keygrip =" and compare it to files found in `~/.gnupg/private-keys-v1.d/`
```
Comment (drduh) [inline README.md]: Spelling.

File: README.md [hunk 4]
```diff
@@ -1201,6 +1202,27 @@ To require a touch for the signing and encrypting keys as well:
...
+
+```
+#!/bin/bash
+{
+    rm -f $HOME/.gnupg/private-keys-v1.d/0C558EE519B341230EF65701417EA4F28923C8A9.key
```
Comment (drduh) [inline README.md]: These files will only exist for you?

**Comment by EliasGabrielsson** [comment]
Went over the spelling and added a shorter explanation of stub keys.

**Comment by drduh** [comment]
I'm not seeing an updated snapshot; there still exist spelling and grammar errors in your change.

**Comment by NicolasRitouet** [comment]
that would be great to have this merged.
@EliasGabrielsson can you fix the errors and conflicts?
@drduh still interested in the PR?

**Comment by drduh** [comment]
Please send a new PR with the desired changes. Thanks.

**Pull request closed without merging**